### ISSUE TITLE
Seeking Assistance with TEE VM Attack Experiment Replication

### ISSUE THREAD

**Issue by pw0rld** [Fraunhofer-AISEC/severed-framework#2]
Hello, Dear author:

I am highly interested in your work involving the replacement of HPA to attack the TEE VM. Recently, I stumbled upon an older version of an SEV machine and I'm eager to replicate your attack experiment.

To begin with, I tried building the severed-framework on my machine. After successfully completing the building steps, I proceeded to set up the TEE VM and executed the '[test.py](https://github.com/Fraunhofer-AISEC/severed-framework/blob/master/test.py)' script. However, I encountered an error during this process. Here is the test log for your reference.

```
[ 1844.760332] [SEVered-1234] Stopped tracking PID 16081
[ 1854.464745] [SEVered-1234] PID 20725 is now being tracked
root@node1 [14:53:50] [master *]
-> # python3 test.py 20725
Setting access id
Success setting access ID

Issuing NMI
Error issuing NMI
```

I kindly request your assistance in providing any information or guidance that would help me replicate this experiment successfully. 
This error, whether verifying this experiment, cannot be replicated on my machine?
Thank you in advance for your support.

**Comment by pw0rld** [comment]
Hello,

I hope this message finds you well. 
I wanted to inform you that I have successfully resolved the previous issue we encountered. 

However, I have a query regarding a different matter. I have designed experiment where I utilize the gdb tool to attach a program in the TEE VM. My objective is to select a specific GVA and retrieve its GPA. 

Following the example provided in the '[test.py](https://github.com/Fraunhofer-AISEC/severed-framework/blob/master/test.py)' script. Subsequently, to achieve my goal of remapping the corresponding PFN, I use the severed_change_mapping_direct function to modify the PFN.

 I aim to remap the corresponding PFN using the command `sb.severed_change_mapping_direct(0x000000000005b32f,0x00000000013e9bad,13391)`.  (I follow the squeen in the '[test.py](https://github.com/Fraunhofer-AISEC/severed-framework/blob/master/test.py)' script and use severed_change_mapping_direct to change the pfn)
![2023-07-06](https://github.com/Fraunhofer-AISEC/severed-framework/assets/14118552/9df89298-f4af-412a-bd15-630f166175f5)
I saw the dmesg message, I guess it changed successfully.

However, when I attempt to read the GVA using gdb, it appears that the changes are not taking effect. The content of the GVA remains unchanged.

I kindly request your assistance in providing any information or guidance on how I can achieve a successful attack scenario. Your support in this matter is greatly appreciated.

Thank you in advance for your attention and support.

Best regards,
Pw0rld
